src/InputForm/InputForm.js

```diff
@@ -0,0 +1,210 @@
+import React from 'react';
+import { NavLink, Route } from 'react-router-dom';
+
+
+// import styling. 
+import './InputForm.scss';
```

Comment:
Components are normally named with an uppercase but other files aren't. i.e. sass partials - these normally live in a styles folder and are preceded by a _